CHANGELOG.md

```diff
@@ -24,11 +24,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Admin LTE 2.0 used. Assets files changed.
-- Move ``sonata_wrapper`` block on ```standard_layout.html.twig```
+- Move ``sonata_wrapper`` block on ``standard_layout.html.twig``
```

Comment:
Sorry to interrupt but: `*.rst` requires 2 backticks and `*.md` requires 1 backtick.

The CHANGELOG is `.md`, so 1 backtick. But please don't make any change in the `.rst` docs of this bundle. We've published them on symfony.com too and we need `.rst` files without errors, so 2 backticks. Thanks!